vendor/vendor.json

```diff
@@ -103,5 +109,5 @@
 			"revisionTime": "2016-07-06T23:03:55Z"
 		}
 	],
-	"rootPath": "github.com/dropbox/dbxcli"
+	"rootPath": "dbxcli"
```

Comment:
Was this an intentional change, or made by the `govendor` too itself? Looking at even the govendor repo, seems like the correct value is what it was before, i.e. `github.com/dropbox/dbxcli` (https://github.com/kardianos/govendor/blob/master/vendor/vendor.json#L30)